ReflectionAnalyzers.Vsix/ReflectionAnalyzers.Vsix.csproj

```diff
@@ -29,6 +26,10 @@
     <None Include="source.extension.vsixmanifest" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.2.3073" />
```

Comment:
📝 The last time I checked, this will not correctly deploy the extension. See https://github.com/DotNetAnalyzers/StyleCopAnalyzers/commit/c643880c885fc0587366654880f2d4e19a87f0d8. I also still needed the import for **Microsoft.VsSDK.targets** in that file. Has something changed between SDK 15.1.192 and 16.2.3073?

Reply:
I'll check.

Reply:
@sharwell You're a lifesaver, thank you! I'll put the imports back, minus the `Label="Paket"`.

I accidentally learned that it seems to work fine to leave<Project Sdk="Microsoft.NET.Sdk">` and only explicitly import the SDK targets, right before the VsSDK targets, but I thought we should play it safe since https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk doesn't confirm that.